### PR TITLE
replace let by const where possible

### PR DESCRIPTION
**Pull request by charbelrami**

### CODE REVIEW

File: web/scripts/userpage.js [hunk 1]
```diff
@@ -256,7 +256,7 @@ friendlyPix.UserPage = class {
                   <div class="fp-pic-text">${text}</div>
               </div>
               <div class="mdl-card mdl-shadow--2dp mdl-cell
-                          mdl-cell--12-col mdl-cell--12-col-tablet mdl-cell--12-col-desktop"
+                          mdl-cell--12-col mdl-cell--12-col-tabconst mdl-cell--12-col-desktop"
```
Comment (nicolasgarnier) [inline web/scripts/userpage.js]: Can you revert this one please? 😆

**Comment by nicolasgarnier** [comment]
Thx for this @charbelrami ! and sorry for the delay!

I'll merge this now and fix the issue myself :)

File: web/scripts/userpage.js (hunk 1 above)
Comment (charbelrami) [inline web/scripts/userpage.js]: :laughing: :sweat_smile:

**Comment by charbelrami** [comment]
Thanks! :)

**Pull request merged**